dcos/cosmospackage.py

```diff
@@ -0,0 +1,53 @@
+from dcos import emitting, http, util
```

Comment:
shouldn't we not merge this commit with the cosmos branch since it has the opposite functionality re need (ie not letting users use the package subcommands if they have cosmos)?